tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/trajopt/config/trajopt_planner_default_config.h

```diff
@@ -95,6 +95,22 @@ struct TrajOptPlannerDefaultConfig : public TrajOptPlannerConfig
   bool smooth_accelerations = true;
   /** @brief If true, a joint jerk cost with a target of 0 will be applied for all timesteps Default: false*/
   bool smooth_jerks = true;
+
+  /** @brief Error function that is set as a constraint for each timestep.
+   *
+   * This is a vector of std::pair<Error Function, Error Function Jacobian>, the error function is required, but the
+   * jacobian is optional (nullptr).
+   *
+   * Error Function:
+   *   arg: VectorXd will be all of the joint values for one timestep.
+   *   return: VectorXd of violations for each joint. Anything > 0 will be a violation
```

Comment:
See other comment

Reply:
I changed it to anything != 0 will be a violation.